GameServerLib/Logic/Packets/PacketHandlers/Handlers/HandleStartGame.cs

```diff
@@ -56,6 +56,11 @@ public bool HandlePacket(Peer peer, byte[] data)
                              * of updating a champion's level, but it works */
                             var levelUpPacket = new LevelUp(player.Item2.Champion);
                             _game.PacketHandlerManager.sendPacket(peer, levelUpPacket, Channel.CHL_S2C);
+                            if (_game.IsPaused)
```

Comment:
Why is the addition here necessary? Isn't the packet sent in any case further below?